modules/acknowledgements/README.md

```diff
@@ -1 +1,7 @@
 The purpose of the Acknowledgement Generator is to aid in dynamically generating an appropriate list of people to acknowledge for any given publication.
+
+The code for the public side of things can be found within `htdocs/acknowledgements/acknowledgements.php`.
+
+Example public facing link: `https://<YOUR-BASE-STUDY-URL>/acknowledgements/acknowledgements.php?date=2017-11-17`
+
+Given the publication date of `2017-11-17`, this will limit the list to people who started on or before `2017-11-17`.
```

Comment:
How is the list of Authors generated? Is it from the Real name of the users or a distinct list store somewhere else?